src/Avatar.ts

```diff
@@ -15,12 +15,21 @@ limitations under the License.
 */
 
 import { RoomMember, User, Room, ResizeMethod } from "matrix-js-sdk/src/matrix";
+import { useIdColorHash } from "@vector-im/compound-web";
 
 import DMRoomMap from "./utils/DMRoomMap";
 import { mediaFromMxc } from "./customisations/Media";
 import { isLocalRoom } from "./utils/localRoom/isLocalRoom";
 import { getFirstGrapheme } from "./utils/strings";
 
+/**
+ * Hardcoded from the Compound colors.
+ * Shade for background as defined in the compound web implementation
+ * https://github.com/vector-im/compound-web/blob/main/src/components/Avatar
+ */
+const AVATAR_BG_COLORS = ["#e9f2ff", "#faeefb", "#e3f7ed", "#ffecf0", "#ffefe4", "#e3f5f8", "#f1efff", "#e0f8d9"];
+const AVATAR_TEXT_COLORS = ["#043894", "#671481", "#004933", "#7e0642", "#850000", "#004077", "#4c05b5", "#004b00"];
```

Comment:
Do we have a future plan to use a Compound component for this, or is it not possible here?

Reply:
Yes, we can. But the design tokens does not export types yet, and TypeScript is shouting at me

<img width="1204" alt="Screenshot 2023-08-30 at 11 20 45" src="https://github.com/matrix-org/matrix-react-sdk/assets/769871/7ade66c2-b5b7-4a7d-accf-f7c11ad03487">

Reply:
So you're planning to fix this with a new Compound release?

Reply:
Yes, pretty much have to unfortunately. I need to export types on that package